File: programas_java/IOT/ClienteDP.py

class ClienteDP:

    # Constructores
    def __init__(this):
        this.nocta  = ""
        this.nombre = ""
        this.tipo   = ""
        this.saldo  = 0

    def __init__(this, datos):
        if(datos == ""):
            this.nocta  = ""
            this.nombre = ""
            this.tipo   = ""
            this.saldo  = 0
        else:
            st = datos.split("_")
            
            this.nocta  = st[0]
            this.nombre = st[1]
            this.tipo   = st[2]
            this.saldo  = int(st[3])

    # Accesors (geters)
    def getNocta(this):
        return this.nocta

    def geNombre(this):
        return this.nombre

    def getTipo(this):
        return this.tipo

    def getSaldo(this):
        return this.saldo

    # Mutators (seters)
    def setNocta(this,ncta):
        this.nocta = ncta

    def setNombre(this,name):
        this.nombre = name

    def setTipo(this,tcta):
        this.tipo = tcta

    def setSaldo(this,cantidad):
        this.saldo = cantidad

    # Metodos
    def toString(this):
        return this.nocta+"_"+this.nombre+"_"+this.tipo+"_"+str(this.saldo)

    def toStringSql(this):
        return "'"+this.nocta+"','"+this.nombre+"','"+this.tipo+"','"+str(this.saldo)+"'"

File: bancoJsp/WEB-INF/classes/ClienteDP.java
package modelBanco;

import java.util.StringTokenizer;

public class ClienteDP
{
	// Atributos
	private String nocta, nombre, tipo;
	private int saldo;

	// Constructores
	public ClienteDP(){
		this.nocta = "";
		this.nombre = "";
		this.tipo = "";
		this.saldo = 0;
	}

	public ClienteDP(String datos){
		StringTokenizer st = new StringTokenizer(datos, "_");
		
		this.nocta = st.nextToken();
		this.nombre = st.nextToken();
		this.tipo = st.nextToken();
		this.saldo = Integer.parseInt(st.nextToken());
	}

	// Accesors (geters)
	public String getNocta(){
		return this.nocta;
	}

	public String getNombre(){
		return this.nombre;
	}

	public String getTipo(){
		return this.tipo;
	}

	public int getSaldo(){
		return this.saldo;
	}

	// Mutators (seters)
	public void setNocta(String ncta){
		this.nocta = ncta;
	}

	public void setNombre(String name){
		this.nombre = name;
	}

	public void setTipo(String tcta){
		this.tipo = tcta;
	}

	public void setSaldo(int cantidad){
		this.saldo = cantidad;
	}

	public String toString(){
		return nocta + "_" + nombre + "_" + tipo + "_" + saldo;
	}

	public String toStringSQL(){
		return "'" + nocta + "','" + nombre + "','" + tipo + "'," + saldo;
	}

	public String toStringHTML(){
		return "<tr><td>" + nocta + "</ td><td>" + nombre + "</ td><td>" + tipo + "</ td><td>" + saldo + "</ td></ tr>";
	}

	/*public String toStringJSON(){
		return "{nocta:'" + nocta + "',nombre:'" + nombre + "',tipo:'" + tipo + "',saldo:'" + saldo + "'}";
	}*/
	
	/*public String toStringJSON(){
		return "{'nocta':'" + nocta + "','nombre':'" + nombre + "','tipo':'" + tipo + "','saldo':'" + saldo + "'}";
	}*/
	
	public String toStringJSON(){
		return "{\"nocta\":\"" + nocta + "\",\"nombre\":\"" + nombre + "\",\"tipo\":\"" + tipo + "\",\"saldo\":\"" + saldo + "\"}";
	}
}